messages/retrieve.metadata.md

```diff
@@ -106,3 +118,13 @@ Sending request to org (metadata API version %s)
 # spinner.polling
 
 Waiting for the org to respond
+
+# error.Conflicts
+
+The changes that would be retrieved will overwrite
+
+# error.Conflicts.Actions
+
+- Re-run the command with --ignore-conflicts if you want to overwrite the remote changes
```

Comment:
```suggestion
- To overwrite the local changes, rerun this command with the --ignore-conflicts flag.
```

Reply:
@mshanemc , okay? Your original message said to overwrite the remote changes.  But this is a retrieve, so that flag will overwrite _local_ changes, no?